HierarchyComposer/Functions/DatabaseComposer.cs

```diff
@@ -154,6 +154,11 @@ public void ComposeDatabase(IReadOnlyList<HierarchyNode> inputNodes, string outp
             {
                 using var transaction = connection.BeginTransaction();
                 using var cmd = new SQLiteCommand(connection);
+
+                cmd.CommandText =
+                    "CREATE VIRTUAL TABLE AABBs USING rtree(Id, min_x, max_x, min_y, max_y, min_z, max_z)";
+                cmd.ExecuteNonQuery();
```

Comment:
Needs a "why comment", also maybe include a "Why comment" in the HierarchyContext.cs class to why we dont create a AABBs table there any